proposals/csharp-9.0/nullable-reference-types-specification.md

```diff
@@ -275,7 +293,11 @@ The null state of an expression is derived from its form and type, and from the
 
 ### Literals
 
-The null state of a `null` and `default` literals is "maybe default". The null state of any other literal is "not null".
+The null state of a `null` literal is "maybe null". 
```

Comment:
I don't think this is true in the case of `T M<T>() where T : class? => null;`

Reply:
Good catch.